.editorconfig

```diff
@@ -12,3 +12,6 @@ trim_trailing_whitespace=true
 indent_style=space
 indent_size=2
 
+[*.json]
+indent_style=space
+indent_size=2
```

Comment:
```suggestion
indent_size=4
```

Reply:
I might be wrong about this, but was under the impression indent_size for json was prefered to be 4.

Reply:
Oh right, you are right! I thought it was the other way around. Will fix that ;)